### ISSUE TITLE
GET /icons/<domain>/icon.png panics for www.zborovna.sk

### ISSUE THREAD

**Issue by mprasil** [dani-garcia/bitwarden_rs#380]
I'm not sure what is causing it yet, this is what I see in the logs:

```
[2019-02-03 21:24:30][reqwest::async_impl::client][DEBUG] redirecting to GET 'https://www.zborovna.sk/novinky/index.php'
[2019-02-03 21:24:30][reqwest::async_impl::response][DEBUG] Response: '200 OK' for https://www.zborovna.sk/novinky/index.php
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { kind: InvalidDigit }', src/libcore/result.rs:997:5
```

**Comment by mprasil** [comment]
As per discussion with @dani-garcia on our chat, this is most likely due to incorrect character in `sizes` attribute of `<link rel="apple-touch-icon">`  - the site is using "×" (multiplication character) instead of "x" to separate the numbers:

```
<link rel="apple-touch-icon" sizes="76×76" href="/images/touch-icon-ipad.png">
```

**Comment by BlackDex** [comment]
Is fixed in: #381

**Comment by dani-garcia** [comment]
Fixed in #381
